### PR TITLE
breaking: rename Tag Ratio to Attribute Completeness

### LINKED ISSUE #493: Rename Tag Ration Indicator to “attribute completeness”

### PR DESCRIPTION
**Pull request by Gigaszi**
### Corresponding issue
Closes #493

### New or changed dependencies
-

### Checklist
- [x] I have updated my branch to `main` (e.g. through `git rebase main`)
- [x] My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing
- [x] I have commented my code
- [x] I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)
- [x] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CHANGELOG.md)

### CODE REVIEW

File: workers/ohsome_quality_analyst/indicators/attribute_completeness/metadata.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-TagsRatio:
-  name: Tags Ratio
+AttributeCompleteness:
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/indicators/attribute_completeness/metadata.yaml:2]:
```suggestion
attribute-completeness:
```

File: workers/ohsome_quality_analyst/reports/building_report/report.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def __init__(
             indicator_layer=(
                 IndicatorLayer("MappingSaturation", "building_count"),
                 IndicatorLayer("Currentness", "building_count"),
-                IndicatorLayer("TagsRatio", "building_count"),
+                IndicatorLayer("AttributeCompleteness", "building_count"),
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/reports/building_report/report.py:17]:
```suggestion
                IndicatorLayer("attribute-completeness", "building_count"),
```

File: workers/ohsome_quality_analyst/reports/road_report/report.py [hunk 3]
```diff
@@ -15,7 +15,7 @@ def __init__(
             indicator_layer=(
                 IndicatorLayer("MappingSaturation", "major_roads_length"),
                 IndicatorLayer("Currentness", "major_roads_count"),
-                IndicatorLayer("TagsRatio", "major_roads_length"),
+                IndicatorLayer("AttributeCompleteness", "major_roads_length"),
```
Comment (matthiasschaub) [inline workers/ohsome_quality_analyst/reports/road_report/report.py:18]:
```suggestion
                IndicatorLayer("attribute-completeness", "major_roads_length"),
```

File: workers/tests/unittests/test_definitions.py [hunk 4]
```diff
@@ -79,7 +79,7 @@ def test_get_valid_indicators(self):
                 "GhsPopComparisonBuildings",
                 "MappingSaturation",
                 "Currentness",
-                "TagsRatio",
+                "AttributeCompleteness",
```
Comment (matthiasschaub) [inline workers/tests/unittests/test_definitions.py:82]:
```suggestion
                "attribute-completeness",
```

File: workers/ohsome_quality_analyst/definitions.py [hunk 5]
```diff
@@ -167,17 +167,17 @@ class RasterDataset:
...
+    ("AttributeCompleteness", "major_roads_length"),
+    ("AttributeCompleteness", "jrc_health_count"),
+    ("AttributeCompleteness", "jrc_education_count"),
+    ("AttributeCompleteness", "jrc_road_length"),
+    ("AttributeCompleteness", "jrc_airport_count"),
+    ("AttributeCompleteness", "jrc_power_generation_plant_count"),
+    ("AttributeCompleteness", "jrc_cultural_heritage_site_count"),
+    ("AttributeCompleteness", "jrc_bridge_count"),
+    ("AttributeCompleteness", "jrc_mass_gathering_sites_count"),
+    ("AttributeCompleteness", "clc_leaf_type"),
```
[the remark this answers is not in the document]
Comment (Gigaszi) [reply workers/ohsome_quality_analyst/definitions.py:180]: done

File: workers/ohsome_quality_analyst/indicators/attribute_completeness/metadata.yaml (hunk 1 above)
Comment (Gigaszi) [reply workers/ohsome_quality_analyst/indicators/attribute_completeness/metadata.yaml]: done

File: workers/ohsome_quality_analyst/reports/building_report/report.py (hunk 2 above)
Comment (Gigaszi) [reply workers/ohsome_quality_analyst/reports/building_report/report.py]: done

File: workers/ohsome_quality_analyst/reports/road_report/report.py (hunk 3 above)
Comment (Gigaszi) [reply workers/ohsome_quality_analyst/reports/road_report/report.py]: done

File: workers/tests/unittests/test_definitions.py (hunk 4 above)
Comment (Gigaszi) [reply workers/tests/unittests/test_definitions.py]: done

**Pull request merged**